scene/3d/multimesh_instance_3d.cpp

```diff
@@ -49,6 +52,26 @@ Ref<MultiMesh> MultiMeshInstance3D::get_multimesh() const {
 	return multimesh;
 }
 
+Array MultiMeshInstance3D::get_meshes() const {
+	if (multimesh.is_null() || multimesh->get_mesh().is_null() || multimesh->get_transform_format() != MultiMesh::TransformFormat::TRANSFORM_3D) {
+		return Array();
+	}
+
+	int count = multimesh->get_visible_instance_count();
+	if (count == -1) {
+		count = multimesh->get_instance_count();
+	}
+
+	Ref<Mesh> mesh = multimesh->get_mesh();
+
+	Array results;
+	for (int i = 0; i < count; i++) {
+		results.push_back(multimesh->get_instance_transform(i));
+		results.push_back(mesh);
```

Comment:
I wonder if it would be better to return a dictionary with a mesh and an array of instance transforms since there can only ever be one mesh.

Alternatively, we could just have a function to return the array of transforms, since the mesh can be obtained already

Looking deeper I see this is so that it can match the CSG API. Honestly, I would add another condition to the VoxelGI code instead of exposing this function. We already have a different case for MeshInstance3D. I would add a new case for MultiMeshInstance3D